content/js/handling-errors.md

```diff
@@ -2,25 +2,47 @@ All asynchronous methods in `skygear` return [Promise](https://www.promisejs.org
 
 ## Basic flow
 
-```js
-skygear.signup('johndoe', 'john.doe@example.com', 'verysecurepassword').then(() => {
+``` javascript
+skygear.signupWithUsername('johndoe', 'verysecurepassword').then(() => {
   // user logged in successfully
 }, (error) => {
-  if (error.code == skygear.Error.DUPLICATED) {
-    console.log('username has been taken');
-  } else {
-    console.log('unexpected error', error.code)
+  if (error.code === 109) {
+    alert('username or email already exists');
```

Comment:
I believe all code examples use `console` for logging? you may want to use `console.error` for this case?